src/test/scala/com/adobe/api/platform/runtime/metrics/PrometheusRecorderTests.scala

```diff
@@ -47,8 +47,13 @@ class PrometheusRecorderTests extends KafkaSpecBase with BeforeAndAfterEach with
       counterStatus(statusMetric, Activation.statusDeveloperError) shouldBe 1
 
       histogramCount(waitTimeMetric) shouldBe 1
+      histogramSum(waitTimeMetric) shouldBe 0.03
```

Comment:
Doing asserts against double at times may fail as they are not accurate. May be use the approach suggested [here][1] and [here][2]

[1]: http://www.scalactic.org/user_guide/Tolerance
[2]: https://stackoverflow.com/questions/27809423/handling-doubles-in-scalatest